core-blocks/paragraph/index.js

```diff
@@ -261,11 +207,12 @@ class ParagraphBlock extends Component {
 						'has-drop-cap': dropCap,
 						[ backgroundColor.class ]: backgroundColor.class,
 						[ textColor.class ]: textColor.class,
+						[ fontSize.class ]: fontSize.class,
```

Comment:
My initial reaction without looking through to see the implementation: Why do we have both `fontSize.class` and `getFontSizeClass`?

Reply:
We use fontSize.class on the edit method because there it was already computed by the HOC. On the save function we don't use the HOC so we use the getFontSizeClass.

Reply:
Seems unnecessarily disjointed.